prov/util/src/util_atomic.c

```diff
@@ -49,11 +49,13 @@ static const size_t ofi_datatype_size_table[] = {
 	[FI_DOUBLE_COMPLEX] = sizeof(ofi_complex_double),
 	[FI_LONG_DOUBLE]    = sizeof(long double),
 	[FI_LONG_DOUBLE_COMPLEX] = sizeof(ofi_complex_long_double),
+	[FI_INT128]  = sizeof(int64_t) * 2,
+	[FI_UINT128] = sizeof(uint64_t) * 2,
```

Comment:
Please add a comment above these that we're calculating the size because int128 may not be supported by the provider.

Reply:
Certainly.